### PR TITLE
chore: Removed no longer used comments from lib/gui/events.ts

### LINKED ISSUE #21753: Remove commented out code that's no longer used in server/lib/gui/events.ts
There's a bunch of code in this file that is commented out. It's unclear why these comments were left here. Let's clean this up. https://github.com/cypress-io/cypress/blob/10.0-release/packages/server/lib/gui/events.ts

Some of this code should be added back however and is related to https://cypress-io.atlassian.net/browse/UNIFY-1226

┆Issue is synchronized with this [Jira Task](https://cypress-io.atlassian.net/browse/UNIFY-1227) by [Unito](https://www.unito.io)
┆author: Jennifer Shehane
┆epic: Chores
┆friendlyId: UNIFY-1227
┆priority: Lowest
┆sprint: Unification Release Backlog
┆taskType: Task

### PR DESCRIPTION
**Pull request by Ashviniv**
- Belongs to https://github.com/cypress-io/cypress/issues/21753

- Closes https://github.com/cypress-io/cypress/issues/21753

### PR Tasks

- [x] Have tests been added/updated?
- [x] Has the original issue (or this PR, if no issue exists) been tagged with a release in ZenHub? (user-facing changes only)
- [x] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [x] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?

### CODE REVIEW

File: packages/server/lib/gui/events.ts [hunk 1]
```diff
@@ -23,141 +23,13 @@ const handleEvent = function (options, bus, event, id, type, arg) {
 
   switch (type) {
     case 'launch:browser':
-      // TIM: Commented out for reference w/ the new implementation
```
Comment (AtofStryker) [inline packages/server/lib/gui/events.ts:26]: @tgriesser do we still need the code commented out here?

Comment (tgriesser) [reply packages/server/lib/gui/events.ts:26]: No, probbaly not

**Review by tgriesser — APPROVED** [review]

File: packages/server/lib/gui/events.ts [hunk 2]
```diff
@@ -23,141 +23,13 @@ const handleEvent = function (options, bus, event, id, type, arg) {
 
   switch (type) {
```
Comment (emilyrohrbough) [inline packages/server/lib/gui/events.ts:24]: @Ashviniv Thank you for this contribution! Looks like removing this leaves this file as a shell/placeholder and it is not actually being leveraged anymore. Can we do the full clean up to remove this file completely?

**Comment by emilyrohrbough** [comment]
Actually, I'm so sorry @Ashviniv, it looks like I have already removed this file in https://github.com/cypress-io/cypress/pull/23002. I didn't realize there was an issue open to clean this up.

**Comment by emilyrohrbough** [comment]
@Ashviniv Would you be interested in picking up this issue instead? https://github.com/cypress-io/cypress/issues/21790

**Pull request closed without merging**